<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComTerm</class>
 <widget class="QMainWindow" name="ComTerm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>201</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>531</width>
    <height>201</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ComTerm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="comPortGB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>141</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>141</height>
         </size>
        </property>
        <property name="title">
         <string>COM port</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>82</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="comPortLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QComboBox" name="portsListComboBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="scanButton">
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectButton">
            <property name="text">
             <string>Connect</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="baudRateGB">
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>141</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>131</width>
          <height>141</height>
         </size>
        </property>
        <property name="title">
         <string>Baud rate</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>123</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="baudRate1200RB">
            <property name="text">
             <string>1200</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="baudRate19200RB">
            <property name="text">
             <string>19200</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="baudRate2400RB">
            <property name="text">
             <string>2400</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="baudRate38400RB">
            <property name="text">
             <string>38400</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="baudRate4800RB">
            <property name="text">
             <string>4800</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="baudRate57600RB">
            <property name="text">
             <string>57600</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="baudRate9600RB">
            <property name="text">
             <string>9600</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QRadioButton" name="baudRate115200RB">
            <property name="text">
             <string>115200</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="dataBitsGB">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>141</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>141</height>
         </size>
        </property>
        <property name="title">
         <string>Data bits</string>
        </property>
        <widget class="QWidget" name="layoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>38</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="dataBiysLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="dataBits5RB">
            <property name="text">
             <string>5</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="dataBits6RB">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="dataBits7RB">
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="dataBits8RB">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="parityGB">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>141</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>141</height>
         </size>
        </property>
        <property name="title">
         <string>Parity</string>
        </property>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>61</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="parityLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="parityNoneRB">
            <property name="text">
             <string>No</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="parityEvenRB">
            <property name="text">
             <string>Even</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="parityOddRB">
            <property name="text">
             <string>Odd</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="paritySpaceRB">
            <property name="text">
             <string>Space</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="parityMarkRB">
            <property name="text">
             <string>Mark</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="stopBitsGB">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>141</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>141</height>
         </size>
        </property>
        <property name="title">
         <string>Stop bits</string>
        </property>
        <widget class="QWidget" name="layoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>48</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="stopBitsLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="stopBits1RB">
            <property name="text">
             <string>1</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="stopBits15RB">
            <property name="text">
             <string>1.5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="stopBits2RB">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="flowControlGB">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>141</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>141</height>
         </size>
        </property>
        <property name="title">
         <string>Flow Control</string>
        </property>
        <widget class="QWidget" name="layoutWidget_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>79</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="flowControlLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="flowControlNoRB">
            <property name="text">
             <string>No</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="flowControlHardRB">
            <property name="text">
             <string>Hardware</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="flowControlSoftRB">
            <property name="text">
             <string>Software</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>37</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish_USA"/>
     <addaction name="actionRussian_Russia"/>
    </widget>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelo">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuView"/>
   <addaction name="menuHelo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRussian_Russia">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Russian (Russia)</string>
   </property>
  </action>
  <action name="actionEnglish_USA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English (USA)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>scanButton</sender>
   <signal>clicked()</signal>
   <receiver>ComTerm</receiver>
   <slot>scanButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ComTerm</receiver>
   <slot>connectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>ComTerm</receiver>
   <slot>tabClose(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scanButtonClicked()</slot>
  <slot>connectButtonClicked()</slot>
  <slot>tabClose(int)</slot>
 </slots>
</ui>
